### PR TITLE
Make table properties case insensitive

### LINKED ISSUE #4024: Table property names should be case insensitive

### PR DESCRIPTION
**Pull request by nileema**
Fixes #4024

### CODE REVIEW

File: presto-main/src/main/java/com/facebook/presto/metadata/TablePropertyManager.java [hunk 1]
```diff
@@ -69,9 +70,10 @@ public void addTableProperties(String catalog, List<PropertyMetadata<?>> tablePr
 
         // Fill in user-specified properties
         for (Map.Entry<String, Expression> sqlProperty : sqlPropertyValues.entrySet()) {
-            PropertyMetadata<?> tableProperty = supportedTableProperties.get(sqlProperty.getKey());
+            String property = sqlProperty.getKey().toLowerCase(ENGLISH);
```
Comment (dain) [inline presto-main/src/main/java/com/facebook/presto/metadata/TablePropertyManager.java]: If we are doing this, we should verify in the `PropertyMetadata` constructor that the property is in lowercase.

Comment (nileema) [inline presto-main/src/main/java/com/facebook/presto/metadata/TablePropertyManager.java]: We do verify that the property is lower case.

Comment (dain) [inline presto-main/src/main/java/com/facebook/presto/metadata/TablePropertyManager.java]: Oh missed that :)

**Comment by dain** [comment]
Looks good

**Pull request merged**